src/guides/v2.3/ext-best-practices/tutorials/send-custom-email.md

```diff
@@ -0,0 +1,81 @@
+---
+group: extension-best-practices
+title: Sending Custom Emails
+contributor_name: Ziffity
+contributor_link: https://www.Ziffity.com/
+---
+
+This topic describes how you can send custom emails in Magento. The below example shows how to send custom emails in Magento.
```

Comment:
Whilst this is a great code fragment for sending emails there is no explanation to the code block, therefore it is not strictly a tutorial. The block and its requirements should be explained. What is this code and why is it there?